Apps/Admin/Client/Pages/SupportPage.razor.cs

```diff
@@ -74,19 +74,19 @@ private UserQueryType SelectedQueryType
 
         private bool MessagingVerificationsLoading => this.MessageVerificationState.Value.IsLoading;
 
-        private bool PhnSelected => this.SelectedQueryType == UserQueryType.Phn;
+        private bool PhnOrDependentSelected => this.SelectedQueryType is UserQueryType.Phn or UserQueryType.Dependent;
```

Comment:
Not sure about this, will it scale if we add more query types? Is it worth spending more time refactoring the queries to a more robust structure?

Reply:
On the front end, Dependent is a Phn in the way the input is displayed and handled. This change is used in one place on the razor page to display the input box label and add the Phn mask for Dependent and Phn. This is a simple change that makes this work for this one additional case we are adding at this time. 
If there are more different types that are going to have very different behavior then a refactor could be in order but not sure that's necessary at this point.